### PR TITLE
[WIP] PROJQUAY-479 - customize /health endpoint

### PR DESCRIPTION
**Pull request by thomasmckay**
Related to: https://issues.redhat.com/browse/PROJQUAY-479

Split the /health endpoint into it's own endpoint and gunicorn process. This will allow it to be run standalone when gunicorn-web is not being run in container.

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -28,6 +28,14 @@ test/__pycache__
...
 **/__pycache__
 static/build/**
+static/fonts/**
+static/webfonts/**
+static/ldn/**
+config_app/static/build/**
+config_app/static/fonts/**
+config_app/static/webfonts/**
+config_app/static/ldn/**
+src/**
```
Comment (thomasmckay) [inline .dockerignore:38]: Allow building container in dir where build artifacts are present.

File: .gitignore [hunk 2]
```diff
@@ -9,6 +10,12 @@ node_modules
 static/ldn
 static/fonts
 static/build
+static/webfonts
+config_app/static/ldn
+config_app/static/fonts
+config_app/static/build
+config_app/static/webfonts
+src
```
Comment (thomasmckay) [inline .gitignore:18]: Allow building container in dir where build artifacts are present.

File: conf/gunicorn_healthy.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+from Crypto import Random
+from util.log import logfile_path
+from util.workers import get_worker_count
+
+
+logconfig = logfile_path(debug=False)
+
+bind = "unix:/tmp/gunicorn_healthy.sock"
+workers = get_worker_count("healthy", 2, minimum=2, maximum=32)
```
Comment (thomasmckay) [inline conf/gunicorn_healthy.py:16]: Too many?

File: .dockerignore (hunk 1 above)
Comment (kurtismullins) [reply .dockerignore:38]: Probably not related to this PR's feature but :+1:

Comment (thomasmckay) [reply .dockerignore:38]: I'll PR this separate but let me know if issues with it or .gitignore.

**Comment by thomasmckay** [comment]
stale

**Pull request closed without merging**